python/py-mapclassify/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-mapclassify
 version             2.6.1
-revision            0
+revision            1
```

Comment:
this change only affects the build dependencies and thus does not warrants a "revision" increase

Reply:
ok, fixed